config/defaults.ini

```diff
@@ -135,7 +135,6 @@ hooks_add =
   70-flatpak-appstream-catalog
   70-flatpak-manifest
   70-ostree-manifest
-  70-packages-manifest
```

Comment:
if it's now unused, please can you delete this file as well

Reply:
Done in 38c0c375df6550c5d7f95bd95ca272d91b2061da